osc/commandline.py

```diff
@@ -8342,6 +8342,36 @@ def do_whois(self, subcmd, opts):
             if len(user) == 3:
                 print("%s: \"%s\" <%s>" % (user[0], user[1], user[2]))
 
+    @cmdln.alias('cpc')
+    def do_createpbuildconfig(self, subcmd, opts, *args):
+        """
+        This command is creating the necessary files to build using pbuild tool.
+        It basically creates _config and _pbuild file in the project directory.
+        Changes from there can not get submitted back, except the project is managed
+        in git.
+
+        Examples:
+            osc cpc REPOSITORY ARCH
+        """
+
+        apiurl = self.get_api_url()
+        if len(args) != 2:
+            raise oscerr.WrongArgs('Wrong number of arguments, require a repository and architecture')
+        repository = args[0]
+        arch = args[1]
```

Comment:
```repository, arch = args```

Reply:
no longer needed, args are handled via argparse and that includes throwing an error if they're not specified